packages/io-fims-common/src/domain/audit-event.ts

```diff
@@ -0,0 +1,41 @@
+import { z } from "zod";
+
+import { userMetadataSchema } from "./user-metadata.js";
+
+export const requestParamsSchema = z.object({
+  client_id: z.string().min(1),
+  redirect_uri: z.string().min(1).optional(),
+  response_type: z.string().min(1).optional(),
```

Comment:
What about using a literal here? If I'm not wrong we only need "code" and "id_token"

```suggestion
  response_type: z.literal("id_token") || z.literal("code"),
```

https://zod.dev/?id=literals

Reply:
@Garma00 you can use or (`||`) in this way! 

you can write 
```js
z.union([z.literal("id_token"), z.literal("code")])
``` 

or just 

```js
z.enum(["id_token", "code"])
```

Reply:
Yes you are right! I have used zod.union instead of || thought

Reply:
🔝